client/src/legacy/LeftAndMain.js

```diff
@@ -499,15 +517,12 @@ $.entwine('ss', function($) {
               }
             });
           }
+          clearButton();
           return false;
         }
 
         self.trigger('submitform', {form: form, button: button});
 
-        // set button to "submitting" state
-        $(button).addClass('btn--loading loading');
-        $(button).prop('disabled', true);
-
         if($(button).is('button')) {
```

Comment:
This condition and the code in this block is now duplicated - does it _need_ to be in both places, or can this second block be removed?

If it does need to be in both places separately, please abstract it to a function call, and add some in-line code comments explaining _why_ it needs to happen both here _and_ above.